content/200-concepts/150-data-platform/08-billing/03-invoices.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: 'Invoices'
+metaTitle: 'Invoices'
+metaDescription: 'The Prisma Data Platform uses Stripe as the payments provider and it is Stripe that issues and sends the monthly invoices when you have a paid plan for a project.'
+---
+
+<TopBlock>
+
+The Prisma Data Platform uses Stripe as the payments provider and it is Stripe that issues and sends the monthly invoices when you have a paid plan for a project.
+
+</TopBlock>
+
+## About invoices and billing cycle
+
+- Billing is done on a monthly basis
+- You are billed and receive invoices for each month on the 1st
+- Each invoice bills upfront the usage until the end of that month
+- If you have multiple projects on a paid plan, you receive a separate invoice for each project
+- The first invoice is a pro-rated charge of the monthly plan price based on the date on which you purchased
+
+## Invoice from initial plan purchase
+
+When you purchase the [`Scale` plan](/concepts/data-platform/billing/plans-and-quotas#scale-plan) for the first time, you will receive an invoice that is at the pro-rated amount for the remainder of the month based on the purchase date.
+
+<!-- Q1: How does this apply to the Enterprise plan -->
+
+For example:
+
+- You purchase the `Scale` plan on the 16th of June
+- We immediately bill you and issue an invoice with the pro-rated (Jun 16 - 30) amount
+- In July, we bill you on the first and issue an invoice with the full amount of the `Scale` plan
+
+![invoicing diagram](pdp-monthly-billing.png)
+
+## Invoices with overages and overage refunds
+
+Paid plans have an unlimited number of team members, environments, and usage of the Prisma Data Proxy, but **they do have quotas**. The quotas limit how many members and environments, and how much Data Proxy usage you can have within the price of the plan.
+
+When you **exceed any of these quotas, you are incurring overages** which we bill at the fixed price in the plan. If you remove the overage (only applies to team members and environments), we include a refund in the invoice for the time during which the overage was not applicable.
```

Comment:
Let's double check on the refunds. Is this really the case?

Reply:
@martzoukos confirmed online that refunds are reconciled with the next month invoice on the 1st. I've now updated all mentions that had this wrong.

Reply:
I +1 the above comment.